charts/cofide-connect/templates/deployment.yaml

```diff
@@ -43,6 +43,20 @@ spec:
           {{- end }}
```

Comment:
On L36, name is {{ .Chart.Name }}. Will that break scripts that do things like

```
kubectl logs --context kind-connect -n connect deployments/cofide-connect-api -c cofide-connect-api
```

Reply:
Yes it will as it stands, I think {{ .Release.Name }} is better (and more consistent)